zerver/openapi/zulip.yaml

```diff
@@ -5979,7 +5979,41 @@ paths:
                 contentType: application/json
       responses:
         "200":
-          $ref: "#/components/responses/SimpleSuccess"
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      detached_files:
+                        type: array
+                        description: |
+                          Returns details on all uploaded files whose only references were removed when editing this message. Users should be presented with this list of messages and asked if they want to delete the uploaded files, which would otherwise remain visible only in message edit history.
```

Comment:
This should be line-wrapped like other API docs are.

Reply:
Whoops didn't catch that in my editor. Fixed.